### DISCUSSION TITLE
How to change "*.worksheet.sc" to not conflict with supercollider?

### DISCUSSION THREAD

**Discussion by w-r-z-k** [scalameta/nvim-metals#508] [Q&A] [answered]
It seems if I open a new worksheet with the file extension ".worksheet.sc", it will recognize it as Scala.

But if I open an existing one, sometimes it recognizes it as a [supercollider](https://en.wikipedia.org/wiki/SuperCollider) program.

I think there's some settings in my neovim setup that recognizes purely on the last dot.  But unfortunately, I cannot find what that might be.  It's likely buried quit deep in the neovim software or a plugin.

Is there a way to set the Scala worksheet extension to, say, ".worksheetsc" (i.e. with out the extra dot)?

Thanks

**Comment by ckipp01** [comment, marked as answer]
Got this fixed upstream. See https://github.com/scalameta/nvim-metals/issues/509 for more details.
